models/labels/ofac/labels_ofac_sanction.sql

```diff
@@ -0,0 +1,45 @@
+{{config(alias='ofac_sanctioned')}}
```

Comment:
I suggest changing this model's name to models/labels/labels_ofac_saction_ethereum.sql